WalletWasabi/Extensions/RPCClientExtensions.cs

```diff
@@ -213,7 +213,7 @@ public static async Task<RpcStatus> GetRpcStatusAsync(this RPCClient rpc, Cancel
 			return newEstimations;
 		}
 
-		/// <returns>(allowed, reject-reason)</returns>
+		/// <returns>Return (Allowed, reject-reason)</returns>
```

Comment:
This comment does not say anything that the code doesn't, so it should be removed.

Reply:
Done.